tests/integration/replication-3.tcl

```diff
@@ -55,6 +55,63 @@ start_server {tags {"repl external:skip"}} {
             after 6000
             r -1 dbsize
         } {0}
+
+        test {Writable replica doesn't return expired keys} {
+            r -1 select 5
+            assert {[r -1 dbsize] == 0}
+            r -1 config set slave-read-only no
+            r -1 debug set-active-expire 0
+            r -1 set key1 5 px 10
+            r -1 set key2 5 px 10
```

Comment:
same about the writes, don't we need to do them on the master (and then `wait_for_ofs_sync` or something)